.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,21 @@
+name: .NET
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: windows-latest

```

Comment:
I noticed one of the source-indexer.sln projects runs on .NET 4.6.2 hence I chose Windows instead of Ubuntu.